INCode/diagramconfiguration.py

```diff
@@ -107,28 +122,57 @@ def reveal_children(self):
             child_tree_item = CallableTreeItem(child, current_item)
             child_tree_item.setExpanded(True)
 
+    def init_preview(self):
+        if self.svg_view_.isVisible():
+            content = self.generate_uml()
+            if content:
+                self.load_view_signal.emit(content)
+
     def export(self):
-        content = self.entry_point_item_.export()
-        self.generate_uml(content)
+        print('exporting ', self.entry_point_item_.callable.name)
+        content = self.generate_uml()
+        self.load_svg_view(content)
```

Comment:
Is `content` always non-`None` at this point, unlike in `init_preview`?

Reply:
It's possible that `content` is `None`  there, but that doesen't really matter because the `load_svg_view` function checks if the `content` is `None`. In `init_preview` I had to check it because the `load_view_signal.emit` doesen't allow a `NoneType`.